scania-digital-design-system/tegel
# fix(toast): content width and close button margins

Removed the content ellipsis so header and subheader can both be unlimitedly long. Also adjusted the margin of the close button to align with component in Figma.

**Solving issue**  
Fixes: [CDEP-3081](https://tegel.atlassian.net/browse/CDEP-3081)

**How to test**  
1. Go to Storybook link below.
2. Check in Toast component.
3. Check that header and subheader can be very long without breaking into an ellipsis or pushing the text outside the component.
4. Compare with component in Figma to make sure it's aligned.

**Checklist before submission**
